resources/js/app.js

```diff
@@ -1,3 +1,30 @@
+import Editor from '@toast-ui/editor'
+import 'codemirror/lib/codemirror.css';
+import '@toast-ui/editor/dist/toastui-editor.css';
+
+const editor = new Editor({
+    el: document.querySelector('#editor'),
+    height: '400px',
+    initialEditType: 'markdown',
+    placeholder: 'Write something cool!',
+})
+
+if (document.querySelector('#createNews')) {
```

Comment:
Styling of your Javascript files is a mess. Please look into our eslint in other projects like here https://github.com/blumilksoftware/toby/blob/main/.eslintrc.js

Reply:
I know of a eslint but for this project is not setup a eslint . A separate pull req for me to configure the eslint

Reply:
Then fix styling here manually, please.